> /* FLEXPWM not routed to LED on this EVK */
> &flexpwm2_pwm3 {
> 	status = "disabled";
> };

This change resulted in a build error: 

FAILED: CMakeFiles/app.dir/src/main.c.obj
ccache /home/maxin/zephyr-sdk-0.16.0/arm-zephyr-eabi/bin/arm-zephyr-eabi-gcc -DBOARD_FLASH_SIZE="CONFIG_FLASH_SIZE*1024" -DCPU_MIMXRT1062DVL6A -DKERNEL -DNDEBUG -DXIP_BOOT_HEADER_DCD_ENABLE=1 -DXIP_BOOT_HEADER_ENABLE=1 -DXIP_EXTERNAL_FLASH -D__PROGRAM_START -D__ZEPHYR__=1 -I/home/maxin/zephyrproject/zephyr/include -I/home/maxin/zephyrproject/build/zephyr/include/generated -I/home/maxin/zephyrproject/zephyr/soc/arm/nxp_imx/rt -I/home/maxin/zephyrproject/modules/hal/cmsis/CMSIS/Core/Include -I/home/maxin/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/devices/MIMXRT1062 -I/home/maxin/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/devices/MIMXRT1062/drivers -I/home/maxin/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/devices/MIMXRT1062/xip -I/home/maxin/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/drivers/common -I/home/maxin/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/drivers/lpuart -I/home/maxin/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/drivers/igpio -I/home/maxin/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/drivers/cache/armv7-m7 -I/home/maxin/zephyrproject/zephyr/modules/hal_nxp/. -isystem /home/maxin/zephyrproject/zephyr/lib/libc/minimal/include -isystem /home/maxin/zephyr-sdk-0.16.0/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/12.2.0/include -isystem /home/maxin/zephyr-sdk-0.16.0/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/12.2.0/include-fixed -fno-strict-aliasing -Os -imacros /home/maxin/zephyrproject/build/zephyr/include/generated/autoconf.h -ffreestanding -fno-common -g -gdwarf-4 -fdiagnostics-color=always -mcpu=cortex-m7 -mthumb -mabi=aapcs --sysroot=/home/maxin/zephyr-sdk-0.16.0/arm-zephyr-eabi/arm-zephyr-eabi -imacros /home/maxin/zephyrproject/zephyr/include/zephyr/toolchain/zephyr_stdint.h -Wall -Wformat -Wformat-security -Wno-format-zero-length -Wno-pointer-sign -Wpointer-arith -Wexpansion-to-defined -Wno-unused-but-set-variable -Werror=implicit-int -fno-pic -fno-pie -fno-asynchronous-unwind-tables -fno-reorder-functions --param=min-pagesize=0 -fno-defer-pop -fmacro-prefix-map=/home/maxin/zephyrproject/zephyr/samples/basic/fade_led=CMAKE_SOURCE_DIR -fmacro-prefix-map=/home/maxin/zephyrproject/zephyr=ZEPHYR_BASE -fmacro-prefix-map=/home/maxin/zephyrproject=WEST_TOPDIR -ffunction-sections -fdata-sections -std=c99 -nostdinc -MD -MT CMakeFiles/app.dir/src/main.c.obj -MF CMakeFiles/app.dir/src/main.c.obj.d -o CMakeFiles/app.dir/src/main.c.obj -c /home/maxin/zephyrproject/zephyr/samples/basic/fade_led/src/main.c
In file included from /home/maxin/zephyrproject/zephyr/include/zephyr/toolchain/gcc.h:92,
                 from /home/maxin/zephyrproject/zephyr/include/zephyr/toolchain.h:50,
                 from /home/maxin/zephyrproject/zephyr/include/zephyr/kernel_includes.h:19,
                 from /home/maxin/zephyrproject/zephyr/include/zephyr/kernel.h:17,
                 from /home/maxin/zephyrproject/zephyr/samples/basic/fade_led/src/main.c:12:
/home/maxin/zephyrproject/zephyr/include/zephyr/device.h:84:41: error: '__device_dts_ord_64' undeclared here (not in a function); did you mean '__device_dts_ord_14'?
   84 | #define DEVICE_NAME_GET(dev_id) _CONCAT(__device_, dev_id)
      |                                         ^~~~~~~~~
/home/maxin/zephyrproject/zephyr/include/zephyr/toolchain/common.h:132:26: note: in definition of macro '_DO_CONCAT'
  132 | #define _DO_CONCAT(x, y) x ## y
      |                          ^
/home/maxin/zephyrproject/zephyr/include/zephyr/device.h:84:33: note: in expansion of macro '_CONCAT'
   84 | #define DEVICE_NAME_GET(dev_id) _CONCAT(__device_, dev_id)
      |                                 ^~~~~~~
/home/maxin/zephyrproject/zephyr/include/zephyr/device.h:210:37: note: in expansion of macro 'DEVICE_NAME_GET'
  210 | #define DEVICE_DT_NAME_GET(node_id) DEVICE_NAME_GET(Z_DEVICE_DT_DEV_ID(node_id))
      |                                     ^~~~~~~~~~~~~~~
/home/maxin/zephyrproject/zephyr/include/zephyr/device.h:227:34: note: in expansion of macro 'DEVICE_DT_NAME_GET'
  227 | #define DEVICE_DT_GET(node_id) (&DEVICE_DT_NAME_GET(node_id))
      |                                  ^~~~~~~~~~~~~~~~~~
/home/maxin/zephyrproject/zephyr/include/zephyr/drivers/pwm.h:258:24: note: in expansion of macro 'DEVICE_DT_GET'
  258 |                 .dev = DEVICE_DT_GET(DT_PWMS_CTLR_BY_IDX(node_id, idx)),       \
      |                        ^~~~~~~~~~~~~
/home/maxin/zephyrproject/zephyr/include/zephyr/drivers/pwm.h:326:34: note: in expansion of macro 'PWM_DT_SPEC_GET_BY_IDX'
  326 | #define PWM_DT_SPEC_GET(node_id) PWM_DT_SPEC_GET_BY_IDX(node_id, 0)
      |                                  ^~~~~~~~~~~~~~~~~~~~~~
/home/maxin/zephyrproject/zephyr/samples/basic/fade_led/src/main.c:17:44: note: in expansion of macro 'PWM_DT_SPEC_GET'
   17 | static const struct pwm_dt_spec pwm_led0 = PWM_DT_SPEC_GET(DT_ALIAS(pwm_led0));
      |                                            ^~~~~~~~~~~~~~~
[67/162] Building C object zephyr/arch/arch/arm/core/aarch...Files/arch__arm__core__aarch32__mpu.dir/arm_core_mpu.c.obj
ninja: build stopped: subcommand failed.

--- a/boards/arm/mimxrt1060_evk/mimxrt1060_evk.dts
+++ b/boards/arm/mimxrt1060_evk/mimxrt1060_evk.dts
@@ -16,6 +16,7 @@
 
 	aliases {
 		led0 = &green_led;
+		pwm-led0 = &green_pwm_led;
 		sw0 = &user_button;
 		kscan0 = &kscan_input;
 		watchdog0 = &wdog0;
@@ -47,6 +48,13 @@
 		green_led: led-1 {
 			gpios = <&gpio1 9 GPIO_ACTIVE_LOW>;
 			label = "User LED1";
+		};
+	};
+
+	pwmleds {
+		compatible = "pwm-leds";
+		green_pwm_led: green_pwm_led {
+			pwms = <&flexpwm2_pwm3 0 PWM_MSEC(20) PWM_POLARITY_NORMAL>;
 		};
 	};
 
@@ -225,6 +233,12 @@ arduino_i2c: &lpi2c1 {
 
 zephyr_udc0: &usb1 {
 	status = "okay";
+};
+
+&flexpwm2_pwm3 {
+	status = "okay";
+	pinctrl-0 = <&pinmux_flexpwm2>;
+	pinctrl-names = "default";
 };
 
 &usdhc1 {

--- a/boards/arm/mimxrt1060_evk/mimxrt1060_evkb.dts
+++ b/boards/arm/mimxrt1060_evk/mimxrt1060_evkb.dts
@@ -6,6 +6,11 @@
 
 #include "mimxrt1060_evk.dts"
 
+/* FLEXPWM not routed to LED on this EVK */
+&flexpwm2_pwm3 {
+	status = "disabled";
+};
+
 &green_led {
 	gpios = <&gpio1 8 GPIO_ACTIVE_LOW>;
 	label = "User LED1";

